lib/Core/SpecialFunctionHandler.cpp

```diff
@@ -324,6 +327,40 @@ void SpecialFunctionHandler::handleReportError(ExecutionState &state,
 				 readStringAtAddress(state, arguments[3]).c_str());
 }
 
+void SpecialFunctionHandler::handleOpenMerge(ExecutionState &state,
+    KInstruction *target,
+    std::vector<ref<Expr> > &arguments) {
+  if (!UseBoundedMerge) {
+    return;
```

Comment:
I would add a `klee_warning` here saying that `klee_open_merge` is ignored as `use-bounded-merge` is not set.